# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/trunk/webrtc/webrtc.gni
# Commit: 3f0981f37c1c
# Full Hash: 3f0981f37c1c499ad52016e526239b390703ba2a
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2019-04-17 21:47:29
# Regressor Bug: 1496359
# File Overlap Count: 1
# Description:
#   Bug 1496359 - Add PipeWire support to desktop capture; r=ng
#   
#   This is an import of upstream commit 318da51f99f91e3de1192f29d7f1824958f9f13e.
#   
#   Depends on D27367
# ==============================================================================

diff -r ceed4e0b3373 -r 3f0981f37c1c media/webrtc/trunk/webrtc/webrtc.gni
--- a/media/webrtc/trunk/webrtc/webrtc.gni	Mon Apr 15 17:12:31 2019 +0000
+++ b/media/webrtc/trunk/webrtc/webrtc.gni	Mon Apr 15 17:26:27 2019 +0000
@@ -9,6 +9,7 @@
 import("//build/config/features.gni")
 import("//build/config/mips.gni")
 import("//build/config/sanitizers/sanitizers.gni")
+import("//build/config/sysroot.gni")
 import("//build/config/ui.gni")
 import("//build_overrides/build.gni")
 
